x-pack/plugins/uptime/public/components/monitor/synthetics/waterfall/components/waterfall_chart.tsx

```diff
@@ -100,13 +102,24 @@ export const WaterfallChart = ({
     return darkMode ? EUI_CHARTS_THEME_DARK.theme : EUI_CHARTS_THEME_LIGHT.theme;
   }, [darkMode]);
 
+  const divRef = useRef<HTMLDivElement | null>(null);
```

Comment:
I'd love to see this have a more semantic name referring to the component the ref contains.